recipes/cdat_info/build.sh

```diff
@@ -0,0 +1,214 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+export GIT_DESCRIBE=`git describe --tags`
+
+cat > cdat_info.py.in << EOF
```

Comment:
Will do.  Many Python packages using UV-CDAT has `import cdat_info`.  I thought I would start with this package to learn about conda-forge.  I will work on a better way to to code this script.  Thanks for your help!